watertap/examples/case_studies/municipal_treatment/tests/test_municipal_treatment.py

```diff
@@ -39,4 +39,5 @@ def test_municipal_treatment():
     assert value(m.fs.recharge_pump.properties[0].flow_mass_comp['toc']) == pytest.approx(9.477e-4, rel=1e-3)
     assert value(m.fs.recharge_pump.properties[0].flow_mass_comp['tss']) == pytest.approx(1.656e-6, rel=1e-3)
 
-    # assert value(m.LCOW) == pytest.approx(VALUE, rel=1e-3)  # TODO: add test for LCOW
+    assert value(m.fs.costing.LCOW) == pytest.approx(5.1533e-7, rel=1e-3) # in M$/m**3
```

Comment:
Does that mean LCOW is displayed in M$/m3 in display_costing? Shouldn't hold anything up, but this would be strange to see for the user initially since LCOW is always reported in $/m3 (and usually never exceeds a few dollars per m3 in extreme cases).

Reply:
No, I explicitly convert it in `display_costing`. The `Expression` itself is in M$/m^3 -- maybe that should be changed in the zero order costing package.